client/pkg/eventlog/lookup.go

```diff
@@ -74,10 +81,15 @@ func (w *ReadableSegmentsWatcher) Start() {
 	go w.Watcher.Run()
 }
 
-func WatchReadableSegments(log *eventlog) *ReadableSegmentsWatcher {
+func WatchReadableSegments(l *eventlog) *ReadableSegmentsWatcher {
 	ch := make(chan []*record.Segment, 1)
 	w := primitive.NewWatcher(defaultWatchInterval, func() {
-		rs, err := log.nameService.LookupReadableSegments(context.Background(), log.cfg.ID)
+		rs, err := l.nameService.LookupReadableSegments(context.Background(), l.cfg.ID)
+		log.Debug(context.Background(), "lookup writable logs", map[string]interface{}{
```

Comment:
how to distinguish error log?

Reply:
I deleted this log to avoid printing too much